<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PastebinSettings</class>
 <widget class="QWidget" name="PastebinSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>228</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout_2">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="nameTitleLabel">
       <property name="text">
        <string>Name / title</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="nameTitleLineEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="exposureLabel">
       <property name="text">
        <string>Exposure</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="exposureComboBox">
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>Public</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Unlisted</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Private (members only)</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="languageLabel">
       <property name="text">
        <string>Language</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="languageComboBox"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="expirationLabel">
       <property name="text">
        <string>Expiration</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="expirationComboBox">
       <item>
        <property name="text">
         <string>10 minutes</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 hour</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 day</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 week</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 weeks</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1 month</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Never</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="loginGroupBox">
     <property name="title">
      <string>Paste as registered user</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="usernameLabel">
          <property name="text">
           <string>Username</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="usernameLineEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Password is not saved. It will be asked during the paste.</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
